src/hotspot/share/gc/shenandoah/shenandoahHeap.hpp

```diff
@@ -698,6 +707,10 @@ class ShenandoahHeap : public CollectedHeap {
 // ---------- Allocation support
 //
 private:
+  // How many bytes to transfer between old and young after we have finished recycling collection set regions?
```

Comment:
General comment (no action needed at this time) about region counts. It appears to me that when talking about quantities that might take positive (`surplus`) or negative (`deficit`) values that are suitably bounded, it might make sense, where possible, to have a single variable of type `ssize_t` and make use of signs for that one variable. Is this such an opportunity?